onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -117,22 +122,22 @@ Status SequentialExecutor::Execute(const SessionState& session_state, const std:
       compute_status = p_op_kernel->Compute(&op_kernel_context);
     } catch (const NotImplementedException& ex) {
       std::ostringstream ss;
-      ss << "Exception was thrown while running Node: " << p_op_kernel->Node().Name()
+      ss << "NotImplementedException was thrown while running Node " << NodeIdForLoggingPurpose(p_op_kernel)
          << ". Error Message: " << ex.what();
       const auto msg_string = ss.str();
       LOGS(logger, ERROR) << msg_string;
       return Status(ONNXRUNTIME, NOT_IMPLEMENTED, msg_string);
```

Comment:
Could use ORT_MAKE_STATUS for a little bit more convenience. 

auto status = ORT_MAKE_STATUS(ONNXRUNTIME, NOT_IMPLEMENTED, "NotImplementedException was thrown while running Node ", NodeIdForLoggingPurpose(p_op_kernel), ". Error Message: ", ex.what());
LOGS(logger, ERROR) << status.ErrorMessage();
return status;